notebooks/official/model_evaluation/custom_tabular_regression_model_evaluation.ipynb

```diff
@@ -0,0 +1,2261 @@
+{
```

Comment:
<p>The following lists should be have each item with a separate line:</p><p> </p><p>Shapley - <em>Note</em>, not recommended for image data -- can be very long running &nbsp;  - XRAI &nbsp;  - Integrated Gradients </p><p>Use Cases: &nbsp;  - Classification and regression on tabular data. </p><p>Use Cases: &nbsp;  - Classification and regression on tabular data. &nbsp;  - Classification on image data. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/942/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='63'/>